### ISSUE TITLE
Cases-cited missing hits

### ISSUE THREAD

**Issue by saizai** [freelawproject/courtlistener#780]
These seem to be because you don't have cites for opinions available before the cite was assigned.

E.g.:

1. 
   A. [CA9 15-15117, Nov. 29, 2017](https://www.courtlistener.com/opinion/4447148/first-amendment-coalition-v-usdoj/)
   B. [CA9 15-15117, Aug 25, 2017](https://www.courtlistener.com/opinion/4421674/first-amendment-coalition-v-usdoj/)
   C. [DCD 16-1090, July 11, 2017](https://www.courtlistener.com/opinion/4407911/gerhard-v-federal-bureau-of-prisons/); [duplicate](https://www.courtlistener.com/opinion/4408172/gerhard-v-federal-bureau-of-prisons/)
   cite:  [DCD 14-0403, Jan. 6, 2016, 155 F. Supp. 3d 1](https://www.courtlistener.com/opinion/3167573/sai-v-transportation-security-administration/)
2. 
   A. [DCD 16-1806, June 26, 2017](https://www.courtlistener.com/opinion/4404616/arora-v-buckhead-family-dentistry-inc/)
   B. [DCD 16-0765, Jan. 6, 2017](https://www.courtlistener.com/opinion/4336523/lamb-v-millennium-challenge-corporation/)
   C. [DCD 15-0322, June 13, 2016](https://www.courtlistener.com/opinion/3212623/brodie-v-burwell/)
   cite: [DCD 14-1876, Dec. 15, 2015, 149 F. Supp. 3d 99, 2015 WL 8966920](https://www.courtlistener.com/opinion/3163016/sai-v-department-of-homeland-security/)
3. 
   A. [DCD 15-0266, May 18, 2017](https://www.courtlistener.com/opinion/4392454/philipp-v-federal-republic-of-germany/)
   B. [DCD 12-1130, Dec. 22, 2016](https://www.courtlistener.com/opinion/4333134/loumiet-v-united-states-of-america/)
   C. [DCD 14-0403, Jan. 6, 2016, 155 F. Supp. 3d 1](https://www.courtlistener.com/opinion/3167573/sai-v-transportation-security-administration/)
   D. [DCD 14-1876, Dec. 15, 2015, 149 F. Supp. 3d 99, 2015 WL 8966920](https://www.courtlistener.com/opinion/3163016/sai-v-department-of-homeland-security/)
   cite: [DCD 14-1876, April 16, 2015, 99 F. Supp. 3d 50](https://www.courtlistener.com/opinion/2794408/sai-v-department-of-homeland-security/)
4. [DCD 17-0686, May 4, 2017](https://www.courtlistener.com/opinion/4389012/allied-progress-v-consumer-financial-protection-bureau/)
   cites: [DCD 14-0403, July 7, 2014, 54 F. Supp. 3d 5](https://www.courtlistener.com/opinion/2682021/sai-v-transportation-security-administration/)
5. [DCD 14-0020, July 14, 2016](https://www.courtlistener.com/opinion/4238589/house-v-courter/)
   cites: DCD 14-403, Feb. 6, 2016, 2016 WL 74397 (doesn't exist; I suspect this should be referring to Jan. 6, 2016, 155 F. Supp. 3d)

None of those cites appear on the authorities list for the citing case, nor on cited-by for the cited case.

**Comment by mlissner** [comment]
Yeah, this is a weakness of our approach — if we're missing a citation on a target case, we don't capture the citations to that case. I'm not sure there's a simple solution to this outside of trying to identify citations by case name, but that can be very messy.
